meson_post_install.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import glob
+import os
+import re
+import subprocess
+import sys
+
+if not os.environ.get('DESTDIR'):
+  project_name = sys.argv[1]
+  datadir = sys.argv[2]
+  icondir = os.path.join(datadir, 'icons', 'hicolor')
+
+  name_pattern = re.compile('hicolor_(?:apps)_(?:\d+x\d+|scalable)_(.*)')
+  search_pattern = '/**/hicolor_*'
+
+  [os.rename(file, os.path.join(os.path.dirname(file), name_pattern.search(file).group(1)))
+   for file in glob.glob(icondir + search_pattern, recursive=True)]
+
+  print('Update icon cache...')
+  subprocess.call(['gtk-update-icon-cache', '-f', '-t', '-q', icondir])
```

Comment:
At first I couldn't figure out where you were using this script at all... then I found it in a different commit from the first one that added the script and implemented the original meson migration.

You may or may not be interested in https://mesonbuild.com/Gnome-module.html#gnomepost_install which does require meson 0.57.0 but handles this all for you.

Reply:
Great, this always puzzle me. This is added for gsettings #54 mainly (but refresh icons, etc... is worth too)

makes more sense. Can you provide a PR now meson is merged?

Reply:
Sure.